code/lora/lora.py

```diff
@@ -0,0 +1,298 @@
+##############################
+# WORKS ONLY WITH PYTHON 3.7 #
+##############################
+
+import os
+import time
+#TODO import argparse
+from pathlib import Path
+from SX127x.LoRa import *
+from SX127x.board_config import BOARD
+
+HOME_PATH = Path.home()
+LORA_LOGS = HOME_PATH / 'lora_logs'
+RX_LOG = LORA_LOGS / 'rx.txt' #File, where all received data is stored
+STATUS_LOG = LORA_LOGS / 'status.txt' #File, where this block's behavior is logged
+CONF_FILE = LORA_LOGS / 'conf.txt' #File, where commands for this block are issued by other scripts
+IMAGES = HOME_PATH / 'images'
+THERMAL = HOME_PATH / 'thermal'
+
+#Log files with data to be sent
+#TODO: Argparse or .conf of some sort
+TX_FILES = [
+    HOME_PATH / 'sensor_logs' / 'alpha.txt',
+    HOME_PATH / 'sensor_logs' / 'bravo.txt',
+    HOME_PATH / 'sensor_logs' / 'charlie.txt'
+]
+
+IMAGES_PATH = HOME_PATH / 'images'
+THERMAL_PATH = HOME_PATH / 'thermal'
+
+UP_PACKET_LENGTH = 11 #Base -> Sat packet length in bytes
+DOWN_PACKET_LENGTH = 25 #Sat -> Base packet length in bytes
+CMD_LENGTH = 10
+
+#DEFAULT transmission mode constants
+MAX_BACKREAD = 5 #Maximal amount of lines expected to be logged into a file within one TX window
+MAX_BUFFER_LENGTH = 50 #Maximal packet buffer length after which it overflows
+MAX_BUFFER_APPEND = MAX_BACKREAD*len(TX_FILES)
+TX_LINE_LENGTH = DOWN_PACKET_LENGTH-1
+
+TX_TIME = 0.11
+RX_TIME = 0.14
+
+THERMAL_HEIGHT = 32
+THERMAL_WIDTH = 24
+
+IMG_QUAL = {
+    'L': 'low',
+    'M': 'medium',
+    'H': 'high',
+    'U': 'ultra'
+}
+
+
+class HEADER: #Packet Headers
```

Comment:
This looks like [dataclass](https://github.com/ericvsmith/dataclasses), it's cool because it generates repr under the hood and other useful stuff